polaris.shopify.com/src/pages/components/[category]/[component].tsx

```diff
@@ -11,18 +12,55 @@ import {
 } from "../../../utils/various";
 import fs from "fs";
 import path from "path";
-import MaxPageWidthDiv from "../../../components/MaxPageWidthDiv";
-import ComponentsPage from "../../../components/ComponentsPage";
-import Nav from "../../../components/Nav";
-import componentsMeta from "../../../data/components.json";
 import { NavItem } from "../../../components/Nav/Nav";
 import NavContentTOCLayout from "../../../components/NavContentTOCLayout";
+import LeftNavWithTOCLayout from '../../../components/LeftNavWithTOCLayout'
 
 interface Props {
   name: string;
   readme: string;
 }
 
+const navItems: NavItem[] = getComponentNav();
+
+
+type NextPageWithLayout = NextPage & Props & {
+  getLayout?: (page: ReactElement) => ReactNode
+}
+
+const ComponentsTwo: NextPageWithLayout = ({ name, readme }) => {
```

Comment:
this is still giving me a typescript error

Reply:
Doing `{ name, readme }: Props` fixed one of the errors but unsure how to fix the other one with NextPageWithLayout and getLayout